dev-libs/libchdr/libchdr-0_p20241211.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+HASH_CHDR=cb077337d53392454e7100a0fd07139ca678e527
+
+DESCRIPTION="Standalone library for reading MAME's CHDv1-v5 formats"
+HOMEPAGE="https://github.com/rtissera/libchdr/"
+SRC_URI="https://github.com/rtissera/libchdr/archive/${HASH_CHDR}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${HASH_CHDR}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=app-arch/zstd-1.5.6
+	>=sys-libs/zlib-1.3.1
+
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/0001-cmake-use-plain-pkg-config-to-discover-zlib-and-libz.patch
+	"${FILESDIR}"/0002-cmake-disable-tests.patch
```

Comment:
Why patch out the tests?

Reply:
The tests build as a binary, that then gets installed. I didn't think that this makes much sense for a "consumer" of the library. Usually there is some flag to disable tests, but nothing here.

Thoughts?

Reply:
Makes sense to me! Seems like an unfortunate way to do the tests, but if that’s how they’re doing it, doesn’t make sense to build their test binary here.

It *is* generally good practice to make sure the tests can run, but that’s probably more effort than it’s worth here, given you’d need to overhaul their build system for them (looks like it doesn’t actually use cmake’s test functionality at all?).